.buildkite/build.rayci.yml

```diff
@@ -0,0 +1,39 @@
+group: build
+steps:
+  - label: ":tapioca: build: wheel {{matrix}}"
+    instance_type: medium
+    commands:
+      - ./ci/build/build-manylinux-ray.sh
+      - ./ci/build/build-manylinux-wheel.sh {{matrix}}
+      - if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then exit 0; fi
+      - export PATH=/opt/python/cp38-cp38/bin:$$PATH
```

Comment:
why is this always cp38-cp38?

Reply:
we just needs any python version actually to run the copy_files.py script; I'm re-thinking all of the logic down below in some later PRs